{{cookiecutter.project_slug}}/home/templates/users/user_list.html

```diff
@@ -0,0 +1,17 @@
+{% raw %}{% extends "base.html" %}
```

Comment:
do we have view for this template?

Reply:
Yes https://github.com/crowdbotics/django-scaffold/blob/master/%7B%7Bcookiecutter.project_slug%7D%7D/users/views.py

Reply:
I am not sure, but for users list don't we need listview. Currently, we just have update and detail view.